### DISCUSSION TITLE
Not able to inference the llm after fine tuning using Auto Train

### DISCUSSION THREAD

**Discussion by abhiramK-171** [huggingface/autotrain-advanced#771] [Q&A]
Hi Abhi,

i have fine tuned a llama llm using auto train, it got executed sucessfully and attached the sample dataset image which i used to fine tune the mode. please find the image below. 

My first Error was : "config.json" this file didnot get generated automatically, i uploaded a manually took from gpt.  image attached below.

after that 

<img width="311" alt="image" src="https://github.com/user-attachments/assets/d7ab404f-4e9b-4d2a-bef0-1702cf87e4b2">

<img width="926" alt="image" src="https://github.com/user-attachments/assets/e9be1ed1-c5f8-4213-b5e4-63a38d8fa80c">
<img width="1280" alt="image" src="https://github.com/user-attachments/assets/1a9ab61e-97b5-4c20-b8c2-a86307428e77">

**Comment by abhishekkrthakur** [comment]
https://huggingface.co/docs/autotrain/faq
